modules/MatchProvider.js

```diff
@@ -5,7 +5,8 @@ import {
 
 class MatchProvider extends React.Component {
   static childContextTypes = {
-    match: matchContextType.isRequired
+    match: matchContextType.isRequired,
```

Comment:
Is there a reason for holding off? I feel like the `Miss` component accessing the `match` context and calling `addMiss` on it could be confusing.